python/src/scipp/utils/dimensions.py

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
+# @author Jan-Lukas Wynen
+
+
+def find_bin_edges(var, ds):
+    """
+    Returns a list of bin-edge dimension labels assuming that
+    var is a coord or attr of ds.
+    """
+    bin_edges = []
+    for idx, dim in enumerate(var.dims):
+        length = var.shape[idx]
```

Comment:
use `var.sizes` to get a dictionary of dims (key) to extents (value). Might simplify this?

Reply:
Good idea! Changed in #2086